python/pyarrow/tests/test_convert_pandas.py

```diff
@@ -164,6 +164,12 @@ def test_strings(self):
         expected = pd.DataFrame({'strings': values * repeats})
         self._check_pandas_roundtrip(df, expected)
 
+    def test_timestamps_notimezone(self):
+        data = {}
+
+        df = pd.DataFrame({'datetime64[ns]': np.array(['2007-07-13', '2006-01-13', '2010-08-13'], dtype='datetime64')})
```

Comment:
Can you add time + milliseconds precision to these sample values (otherwise the time bit is all zero)?